src/sentry/static/sentry/app/utils/marked.jsx

```diff
@@ -54,4 +54,11 @@ marked.setOptions({
   sanitize: true,
 });
 
+const noParagraphRenderer = new marked.Renderer();
```

Comment:
This renderer doesn't have any of the prototype extensions done above. Should it?

Reply:
Looks like this is just previous code but renamed the exported function (`markedSimple` -> `singleLineRenderer`)

Reply:
I can make it use the renderer above in case we end up using this for something with user input